src/github.js

```diff
@@ -62,14 +63,26 @@ const comment = async (opts) => {
 };
 
 const get_runner_token = async () => {
-  const {
-    data: { token }
-  } = await octokit.actions.createRegistrationTokenForRepo({
-    owner,
-    repo
-  });
-
-  return token;
+  if (typeof owner !== 'undefined' && typeof repo !== 'undefined') {
```

Comment:
typeof owner !== 'undefined' does not have to be checked since its a condition of both